src/main/java/cst438flights/controller/CustomerController.java

```diff
@@ -16,14 +18,27 @@
     @Autowired
     private CustomerService customerService;
 
+    @Autowired
+    private CustomerRepository customerRepository;
+
     @PostMapping("/customer")
     public String customerHistory(
             @RequestParam("email") String email,
+            @RequestParam("password") String password,
             Model model) {
 
-        Iterable<FlightInfo> flights = customerService.getPreviousFlights(email);
-        model.addAttribute("flights", flights);
+        Customer customer = customerRepository.findByEmail(email);
```

Comment:
Excellent password solution, tested to work.

Reply:
Should probably hash password if production for security